Google.Api.Gax.Grpc/Rest/RestChannelCredentialsConfigurator.cs

```diff
@@ -0,0 +1,51 @@
+﻿/*
+ * Copyright 2020 Google LLC
+ * Use of this source code is governed by a BSD-style
+ * license that can be found in the LICENSE file or at
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+
+using Grpc.Core;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Google.Api.Gax.Grpc.Rest
+{
+    class RestChannelCredentialsConfigurator : ChannelCredentialsConfiguratorBase
```

Comment:
There's another class named the same and very similar in CredentialExtensions. I believe this one here is not being used, only the one in CredentialExtensions.

Reply:
Deleted.